.github/labeler.yml

```diff
@@ -6,31 +6,224 @@
+#
+# Here quick crash course about the glob pattern:
+# When we have "**/homework/**" returns this always true, if in the path of file
+# a directory "homework" contains:
+# "app/lib/homework/add.dart" -> true
+# "app/lib/timetable/add.dart" -> false
+#
+# When we have "**/**homework**" returns this always true, if a file contains
+# the word "homework":
+# "app/lib/homework_add.dart" -> true
+# "app/lib/fab_homework_button.dart" -> true
+# "app/lib/homeworks_page.dart" -> true
+# "app/lib/homewor_page.dart" -> false
+#
+# Try this the following link to play around with the glob pattern:
+# https://globster.xyz/?q=**%2F**homework**&f=app%2Flib%2Fhomework_page.dart%2Capp%2Flib%2Ftimetable_page.dart%2Capp%2Flib%2Fhomework%2Fadd.dart&embed=1
+#
+# Another recommendation is to keep the global general and wide. For the
+# "feature: homework" label, do not list all the paths to the homework
+# directories. Instead, just use something like "**/homework/**". This will
+# automatically include the app files, test files, package files and asset files
+# that contain a homework directory. Of course, this can lead to false positives
+# in a few cases. However, we prefer the general glob to the small chance of a
+# false positive because we think the chance of not assigning the label with too
+# specific globs is much higher than having a false positive with a general
+# glob. And when searching for a PR with labels, it is usually better to find a
+# few false positives than not to find a PR about a change in the homework
+# feature that does not have the homework label. 
+
 "ci/cd":
   - .github/workflows/**
   - .github/labeler.yml
   - .github/dependabot.yml
   - codemagic.yaml
+  - "**/firebase.json"
+  - "**/.firebaserc"
+  - "bin/check_license_headers.sh"
+  - "bin/add_license_headers.sh"
+  - "bin/source_of_truth/**"
 
 "documentation":
   - "**/*.md"
   - "**/*.mdx"
+  - "docs/**"
+  - "docs.yaml"
 
 "dependencies":
   - "**/pubspec.yaml"
   - "**/pubspec.lock"
+  - ".fvm/**"
 
 "testing":
   - "**/*_test.dart"
+  - "**/test_driver/**"
+  - "**/test/**"
+  - "**/integration_test/**"
+  - "**/dart_test.yaml"
 
 "platform: android":
   - "app/android/**"
+  - "**/**android**"
+  - "**/**_android/**"
+  - "**/android_**/**"
+  - "**/**_android_**/**"
 
 "platform: ios":
   - "app/ios/**"
+  - "**ios**"
+  - "**/**_ios**/**"
+  - "**/ios_**/**"
+  - "**/**_ios_**/**"
 
 "platform: macos":
   - "app/macos/**"
+  - "**/**macos**"
+  - "**/**_macos/**"
+  - "**/macos_**/**"
+  - "**/**_macos_**/**"
 
 "platform: web":
   - "app/web/**"
+  - "**/**web**"
+  - "**/**_web/**"
+  - "**/web_**/**"
+  - "**/**_web_**/**"
+
+"code quality":
+  - "**/analysis_options.yaml"
+  - "lib/sharezone_lints/**"
+
+"legal":
+  - "LICENSE"
+  - "**/**privacy_policy**"
+  - "**/**imprint**"
+
+"sharezone: cli":
+  - "tools/sz_repo_cli/**"
+  - "bin/sz"
+  - "bin/sharezone"
+
+"ui / ux":
+  - "**/theme/**"
+  - "**/**theme**"
+  - app/fonts/**
+  - lib/sharezone_widgets/**
```

Comment:
"TDB"?

Reply:
lol, I'm stupid

Reply:
TBD*

Reply:
to be discussed

Reply:
So I'm mean if all changes in `lib/sharezone_widgets/**` should get the "ui/ux" label because primarily the `sharezone_widgets` package UI widgets contains